### PR TITLE
Remove snapshot validator set total stake amount nanos

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Comment by lazynina** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1318?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1318** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1318?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1313** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1313?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1314](https://github.com/deso-protocol/core/pull/1314) <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1314?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lazynina and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/db_utils.go [hunk 1]
```diff
@@ -558,10 +558,9 @@ type DBPrefixes struct {
 	// Note: we parse the ValidatorPKID from the key and the value is nil to save space.
 	PrefixSnapshotValidatorSetByStakeAmount []byte `prefix_id:"[89]" is_state:"true"`
 
-	// PrefixSnapshotValidatorSetTotalStakeAmountNanos: Retrieve a snapshot of the validator set's total amount of
-	// staked DESO by SnapshotAtEpochNumber.
-	// Prefix, <SnapshotAtEpochNumber uint64> -> *uint256.Int
-	PrefixSnapshotValidatorSetTotalStakeAmountNanos []byte `prefix_id:"[90]" is_state:"true"`
+	// Prefix 90 was previously used for the PrefixSnapshotValidatorSetTotalStakeAmountNanos prefix.
```
Comment (tholonious) [inline lib/db_utils.go:561]:
```suggestion
	// Prefix 90 is deprecated. It was previously used for the PrefixSnapshotValidatorSetTotalStakeAmountNanos prefix.
```

File: lib/db_utils.go [hunk 2]
```diff
@@ -880,9 +879,6 @@ func StatePrefixToDeSoEncoder(prefix []byte) (_isEncoder bool, _encoder DeSoEnco
 	} else if bytes.Equal(prefix, Prefixes.PrefixSnapshotValidatorSetByStakeAmount) {
 		// prefix_id:"[89]"
 		return false, nil
```
Comment (tholonious) [inline lib/db_utils.go:881]: Add a comment here to explain why prefix 90 is missing here too

**Review by tholonious — APPROVED** [review]

**Comment by lazynina** [comment]
### Merge activity

* **May 14, 6:47 PM EDT**: @lazynina started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1318).

**Pull request merged**